platzky/admin/fake_login.py

```
@@ -0,0 +1,35 @@
+def get_fake_login_html() -> str:
+    """Generate HTML for fake login buttons."""
+    html = """
+    <div class="col-md-6 mb-4">
+      <div class="card">
+        <div class="card-header">
+          Development Login
+        </div>
+        <div class="card-body">
+          <p class="text-danger"><strong>Warning:</strong> For development only</p>
+          <div class="d-flex justify-content-around">
+            <a href="{{ url_for('admin.fake_login', role='admin') }}" class="btn btn-primary">Login as Admin</a>
```

Ruff (0.8.2)

Line too long (112 > 100)

(E501)